packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -90,6 +91,8 @@ export function makeRumPublicApi<C extends RumInitConfiguration>(startRumImpl: S
       initConfiguration.clientToken = initConfiguration.publicApiKey
     }
 
+    ;(initConfiguration as InitConfiguration).useAlternateIntakeDomains = true
+
     const { configuration, internalMonitoring } = commonInit(initConfiguration, buildEnv)
```

Comment:
Suggestion: the cast could be avoided like that:

```suggestion
    const { configuration, internalMonitoring } = commonInit(
      { ...initConfiguration, useAlternateIntakeDomains: true },
      buildEnv
    )
```

WDYT?

Reply:
Nice !